baggage/baggage.go

```diff
@@ -596,6 +607,9 @@ func (b Baggage) Len() int {
 
 // String encodes Baggage into a header string compliant with the W3C Baggage
 // specification.
+// It would ignore members where the member key is invalid with the W3C Baggage
+// specification. This could happen for a UTF-8 key, as it may contain
+// invalid characters.
```

Comment:
Is it not repetitive?

```suggestion
// specification.
```